taichi/rhi/metal/metal_device.mm

```diff
@@ -240,7 +240,10 @@
 MetalCommandList::MetalCommandList(const MetalDevice &device,
                                    MTLCommandQueue_id cmd_queue)
     : device_(&device) {
-  cmdbuf_ = [cmd_queue commandBuffer];
+  @autoreleasepool {
```

Comment:
Do you mean that even when we didn't write `@autoreleasepool {` there is a outer release pool already?

Reply:
yes, there is an outer pool by default in the cocoa/UIKit event loop.

Reply:
okok. (That seems like a absolute crap design by Apple)